Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -136,7 +136,9 @@ def _create_alert_query(event_severity_filter, start_time):
 
 def _get_all_high_risk_employees_from_page(page, risk_tags):
     res = []
-    for employee in page["items"]:
+    # Note: page is a `Py42Response` and has no `get()` method.
+    employees = page["items"]
```

Comment:
Feedback from them is that they call out all cases where we are not using `get()`.... Py42Response accesses might through them off but there aren't too many and it's only a minor thing. Hopefully these comments help.